code/game/objects/items/hand_items.dm

```diff
@@ -660,6 +660,8 @@
 	. = ..()
 	var/obj/projectile/ink_spit/ink_spit =  new (target)
 	ink_spit.on_hit(target)
+	if(!QDELETED(ink_spit)) // in case it somehow remains around
+		qdel(ink_spit)
```

Comment:
This whole thing seems kinda cursed???? I'm okay with the fix but why the hell is it like this?